src/rail/creation/degradation/observing_condition_degrader.py

```diff
@@ -333,6 +337,45 @@ def assign_pixels(self, catalog: pd.DataFrame) -> pd.DataFrame:
         catalog = pd.concat([catalog, assigned_pix], axis=1)
 
         return catalog
+    
+    # this is milky way extinction, should be added before other observing conditions is applied
+    def apply_galactic_extinction(self, pixel: int, pixel_cat: pd.DataFrame) -> pd.DataFrame:
+        """
+        MW extinction reddening of the magnitudes
+        """
+        # set the A_lamba/E(B-V) values for the six LSST filters 
```

Comment:
I know this was already in the codebase, but looking at it again, would there be a way to make this flexible enough for other filter specifications, e.g. to forecast the result of combining LSST with another survey?

Reply:
You have to make some assumptions about the dust to compute A/E(B-V), I copied some stuff that Scott Daniel did a long time ago using the (I think now defunct) `lsst.sims.photUtils` package to do it.  I think it's definitely beyond scope here, but in the future I think someone would just have to dig into photUtils and copy some of that code and what I have here: 
https://github.com/LSSTDESC/DC2-analysis/blob/master/contributed/Derive_A_EBV_coefficients.ipynb